content/english/linux-basics/activity-9.md

```diff
@@ -25,17 +30,15 @@ Remove the file we have decided is causing our problems. How can we check that t
 The format is: `rmdir [directoryName]`.
 
 {{% notice warning %}}
-
 `rmdir` permanantly removes the directory.
-
 {{% /notice %}}
 
 How can you remove the directory that the bad file was in?
 
-![rmdir command](../images/Act9.2.PNG?classes=border,shadow)
+![rmdir command](../images/Act9.2.png?classes=border,shadow)
 
 Do you remember the command to remake the directory that we deleted?
 
-![mkdir command](../images/Act9.3.PNG?classes=border,shadow)
+![mkdir command](../images/Act9.3.png?classes=border,shadow)
 
-Now let's go check if we know a command to fix the `secret-message.txt`...
+*Now let's go check if we know a command to restore the `secret-message.txt`...*
```

Comment:
A warning about removing files in general would be nice, since this is (usually) a dangerous operation if we're not removing malware files.

Reply:
Added a warning at line 40.